webbeans-jetty9/pom.xml

```diff
@@ -98,6 +98,12 @@
             <artifactId>openwebbeans-web</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.openwebbeans</groupId>
+            <artifactId>openwebbeans-el22</artifactId>
```

Comment:
not strictly necessary, but the only way I could force invoker to install the artifact and thereby ensure we are testing with what was built (doesn't expose a transitive dependency - though would expose a transitive version preference... but for the version they should be using anyway)